### PR TITLE
fix: db config documentation and test fixes

### PR DESCRIPTION
**Pull request by alanshaw**

### CODE REVIEW

File: packages/api/pw-test.config.cjs [hunk 1]
```diff
@@ -56,8 +56,7 @@ module.exports = {
       stdout.toString().includes('Server started on: http://localhost:9094')
     ) {
       console.log('⚡️ Mock cluster started.')
-      await execa(cli, ['db', '--start', '--project', project])
-      await delay(2000)
+      await execa(cli, ['db', '--init', '--start', '--project', project])
```
Comment (alanshaw) [inline packages/api/pw-test.config.cjs:59]: Added `--init` here - not sure how things were working in CI before...

Comment (hugomrdias) [reply packages/api/pw-test.config.cjs:59]: start seems to be enough run everything need, did it fail for you locally ?

Comment (hugomrdias) [reply packages/api/pw-test.config.cjs:59]: start seems to run everything need, did it fail for you locally ?

**Review by hugomrdias — APPROVED** [review]

File: packages/api/pw-test.config.cjs (hunk 1 above)
Comment (alanshaw) [reply packages/api/pw-test.config.cjs:59]: Ah seems to work now if I remove it so I'll revert.

**Pull request merged**